profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -370,7 +370,7 @@ function _nexteuropa_multilingual_warning_message(&$form, $node) {
   }
   $warning_message_languages = variable_get('nexteuropa_multilingual_warning_message_languages');
   if (!empty($warning_message_languages)) {
-    $message = token_replace($warning_message_languages, array('node' => $node));
+    $message = strip_tags(token_replace($warning_message_languages, array('node' => $node)), '<b><i><strong><em>');
```

Comment:
Is this list of allowed tags come from somewhere and it is self-decision?

It is just to know and I can approve the PR as it is!

Reply:
I don't know if there is a rule for the platform that tells which tags can be used in this kind messages. I looked for different messages in the platform and I realised that only italics and bold tags were used in the messages that I found, so those are the ones I added here.

Reply:
Thx @AlbaParrilla  for your explanation.

Ticket ready for next step.